docs/Researcher/cli-reference/runai-submit.md

```diff
@@ -372,25 +370,25 @@ runai submit --job-name-prefix -i gcr.io/run-ai-demo/quickstart -g 1
 
 > External access type to interactive jobs. Options are:
 >
->  * `nodeport`
->  * `external-url`
+> * `nodeport`
+> * `external-url`
 >
-> This flag can be used both in the standard format which uses command flag for all the options.
+> This flag now supports more than one `service-type`. Multiple service types are supported in CSV style using multiple instances of the same option and commas to separate the values for them. You can also add the optional parameter `custom-url` to the command.
 >
 > For example:
 >
-> `runai submit test-jup -p team-a -i gcr.io/run-ai-demo/jupyter-tensorboard --service-type external-url --port 8888`
+>`runai submit test-np -p team-a -i ubuntu --service-type nodeport,port=30000:7070 --service-type external-url,port=30001`
 >
-> `runai submit test-np -p team-a -i ubuntu --service-type nodeport --port 30000:7070`
+>`runai submit test-np -p team-a -i ubuntu --service-type nodeport,port=30000:7070,port=9090 --service-type external-url, --service-type external-url,port=8080,custom-url=https://my.domain.com/url`
```

Comment:
the second service-type here (out of 3) is invalid!
"--service-type external-url,"